cmd/timoni/main.go

```diff
@@ -114,4 +116,27 @@ func addKubeConfigFlags(cmd *cobra.Command) {
 	kubeconfigArgs.Namespace = &namespace
 
 	cmd.PersistentFlags().StringVarP(kubeconfigArgs.Namespace, "namespace", "n", *kubeconfigArgs.Namespace, "The instance namespace.")
+	cmd.RegisterFlagCompletionFunc("namespace", completeNamespaceList)
+}
+
+func completeNamespaceList(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
```

Comment:
Let’s move this and the instance function to a dedicated file e.g. `completion.go`

Reply:
Moved it to `completion_funcs.go` (a0793c6) instead of `completion.go`, to not collide with the existing `timoni completion` command